README.md

```diff
@@ -1,2 +1,104 @@
 # learn-raspberry-pi-iot
 Learn how to build Internet of Things (IoT) Apps using Raspberry Pi and JavaScript!
+
+<img src="https://www.raspberrypi.org/wp-content/uploads/2012/03/raspberry-pi-logo.png" width="40%" style="display: block; margin: 0 auto;">
+
+## *Why*?
+
+A **[Raspberry Pi](https://vimeo.com/90103691)** (Raspberry Pi) is a 'mini computer' that was created to help people learn about the basics of computers. They are _great_ for resource to get started on learning about hardware as they consist of the basic components of a computer. Raspberry Pi can run on an OS, which means that we can talk to the computer with higher level languages like Javascript!
+
+## *What*?
+**Internet Of Things (IoT)** describes the [internetworking of physical devices](https://en.wikipedia.org/wiki/Internet_of_things)... that basically means that multiple systems (e.g. computers and microprocessors) can talk to each other to do _awesome_ things.
+
+<img src="https://pbs.twimg.com/media/Bl7pVeHCEAAXJxl.jpg" width="80%" style="display: block; margin: 0 auto;">
+
+An example is a 'Smart Home' which might consist of a thermometer, microprocessor, a server and a mobile. The thermometer might _measure_ the temperature of a room; the microprocessor might _read_ the temperature every 5 minutes and _send_ those results to a computer. This computer might then _save_ each measurement and send them to the Home Owner, displaying the measurements in the form of a _graph_ so that they can easily see their energy consumption.
+
+**Raspberry Pi** runs on a Linux OS, called Raspbian.
+
+#### Background links:
+* [Introduction to Raspberry Pi](https://www.raspberrypi.org/help/faqs/#introWhatIs)
```

Comment:
we prefer to have the links in the format:
`{Link description} - {url}`
But in this context it's fine. 👍 
(_no need to change it_)